charts/backingservices/requirements.yaml

```diff
@@ -5,7 +5,7 @@
 # To disable deploying Elasticsearch in SRS, set the 'srs.srsStorage.provisionInternalESCluster' parameter in backingservices values.yaml to false.
 dependencies:
 - name: elasticsearch
-  version: "7.10.2"
+  version: "7.17.3"
```

Comment:
Why is this version 7.1.7.3 instead of 7.17.9? 
Is this intentional?

Reply:
Hi Srini, `7.17.3 is the elastic helm chart version` which is used to spin up the elastic search cluster using helm.
Reference: https://github.com/pegasystems/pega-helm-charts/pull/647